src/web/components/PortalHeader/PortalHeader.tsx

```diff
@@ -36,10 +39,35 @@ export function PortalHeader({
   const { participantId } = useParams();
   const { LoggedInUser } = useContext(CurrentUserContext);
 
-  const routes = [ParticipantInformationRoute, TeamMembersRoute, EmailContactsRoute];
-  if (LoggedInUser?.user?.isUid2Support) {
-    routes.push(AuditTrailRoute);
-  }
+  const [routes, setRoutes] = useState<PortalRoute[]>([
+    ParticipantInformationRoute,
+    TeamMembersRoute,
+    EmailContactsRoute,
+  ]);
+
+  useEffect(() => {
+    const getUserRolesForCurrentParticipant = async () => {
+      const user = LoggedInUser?.user;
+      if (user?.isUid2Support) {
```

Comment:
Is it okay to assume UID2Support is all or nothing for participants - meaning a user with UID2Support either has that role for all their participants or none? If not, I can add a check below to also check if the userRoles that are returned contain `UserRoleId.UID2Support`